Microsoft.Toolkit.Uwp.UI.Controls/ImageCropper/ImageCropper.Logic.cs

```diff
@@ -460,7 +460,7 @@ private void UpdateMaskArea(bool animate = false)
                 case CropShape.Rectangular:
                     if (_innerGeometry is RectangleGeometry rectangleGeometry)
                     {
-                        var to = new Rect(new Point(_startX, _startY), new Point(_endX, _endY));
+                        var to = new Point(_startX, _startY).ToRect(new Point(_endX, _endY));
```

Comment:
@Sergio0694 noticed in this case we don't have a way to construct a rect from x, y, x2, y2. Would it be useful for us to create a helper for this scenario too and avoid creating both two new points just to create a Rect? This pattern is used 3 times in the other file as well. Thoughts?

Reply:
@michael-hawker That would be the **ideal** use case for static extension methods 😢
As in, ideally we'd just add a `Rect.FromCoordinates` static method taking `x1`, `y1`, `x2`, `x2`.
Withough that (C# doesn't support that yet unfortunately), not sure about this - I mean we could add a `RectHelper` class with that method, but having a whole new helper class just for that single method might be a bit overkill 🤔
If you like that idea just let me know and I can create another PR with that though!